src/decays/specializations/H/decay_H_to_ubaru.inc

```diff
@@ -177,7 +177,24 @@ double CLASSNAME::get_partial_width<Higgs, bar<UpTypeQuark>::type, UpTypeQuark>(
 
       result_S = (1-4.*xOS)*result_DR_S + 4*xOS*result_OS_S;
       result_P = (1-4.*xOS)*result_DR_P + 4*xOS*result_OS_P;
-}
+   }
+
+   if (static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::call_higgstools)) ||
+       static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::call_lilith)) ||
+       static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::calculate_normalized_effc))
+   ) {
+      neutral_higgs_effc.add_coupling(
+         field_as_string<Higgs>(indexIn),
+         {
+            boost::hana::unpack(bar<UpTypeQuark>::pdgids, _to_array<bar<UpTypeQuark>::numberOfGenerations>).at(indexOut1.at(0)),
+            boost::hana::unpack(UpTypeQuark::pdgids, _to_array<UpTypeQuark::numberOfGenerations>).at(indexOut2.at(0))
+         },
+         std::pair<std::string const&, std::complex<double>> {
```

Comment:
The first argument of this `pair` is a reference to a string. Do you know whether the string lives long enough when passing the pair to `add_coupling()`?

Reply:
You're right. I wasn't thinking. Done in 7a425d43beebf4ac277b67e30046be3c7fea0909.

Reply:
Thanks for this! Now that it is changed, I'm wondering if `add_coupling` could take the `std::pair` as a `const` references. That could prevent a string copy.

Reply:
Done in 377f66666f5cd3530247cee671575d772a8291fe.